source/documentation/contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,100 @@
+[[contributing-user-stories]]
+= Contribute user stories to oVirt documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about how to create user stories for oVirt GitHub repository
+
+toc::[]
+
+== Modularization backstory
+oVirt docs are modularized.
+All existing content has been replaced with content that is based on user stories and
+complies with the modularization guidelines. All future content must both
```

Comment:
Suggestion: Add a brief description/definition of a user story -- with an example -- here.

Reply:
The existing example is very close in proximity to this sentence...do you really think that's necessary?